exception-lists/tracker-allowlist.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file should follow the format of the other files in this directory. Namely the filename should be `tracker-allowlist-sites.json` and `allowlistedTrackers` should be an array.

Reply:
Why does it need to be an array?

Reply:
That's the current expected format. Changes will need to be made in `index.js` otherwise to support this and we should probably discuss it.

Reply:
Could it go under Settings instead?
```
trackerAllowlist: {
    settings: {
        allowlistedTrackers: {}
    }
}
```